### PR TITLE
feat(docs): docs customer showcase

### PR DESCRIPTION
**Pull request by chdeskur**

### CODE REVIEW

File: fern/pages/docs/getting-started/customer-showcase.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Customer Showcase
+---
+
+At Fern, we help businesses build and distribute world-class documentation that developers love. From startups to global enterprises, our customers trust Fern to elevate their developer experience and _decrease time to_ `200`. Below are some of the companies leveraging Fern to power their documentation.
```
Comment (dannysheridan) [inline fern/pages/docs/getting-started/customer-showcase.mdx:5]: About Fern Docs

Fern Docs is a documentation tool for creating beautiful, customizable developer docs. Style your docs to match your brand, write content in MDX, add your OpenAPI spec, and integrate seamlessly with CI/CD. Below are some companies building with Fern to elevate their developer experience.

Comment (chdeskur) [reply fern/pages/docs/getting-started/customer-showcase.mdx]: i pushed an update that aligns more with that framing

File: fern/docs.yml [hunk 2]
```diff
@@ -354,6 +354,9 @@ navigation:
           - page: Publish your Docs
             path: ./pages/docs/getting-started/publishing-your-docs.mdx
             icon: fa-regular fa-upload
+          - page: Customer Showcase
```
Comment (dannysheridan) [inline fern/docs.yml:357]: why put it this low? isn't this one of the first things you'd want someone evaluating Fern to see?

could see Getting Started just being:
- Overview
- Showcase
- Quickstart

don't have clear guidance on where pages like Global Config and Project Structure would go though

**Review by dannysheridan — APPROVED** [review]

**Comment by dannysheridan** [comment]
in a mergable state! nicely done. only open question is around where in the IA we want to put this page.

**Pull request merged**